docker-start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p /data/db
+mongod &
+mkdir instance
+echo 'DEBUG=False' >> instance/config.py
```

Comment:
If this is meant for development, why do we need to set `DEBUG` to false?

Reply:
Not sure what this `DEBUG` does. @lukpueh should I make it `True` ?

Reply:
It's a toggle to enable/disable [`DEBUG`](mode/http://flask.pocoo.org/docs/0.12/quickstart/#debug-mode), which show additional debugging information and should be disabled in production. 
I have set it to [`TRUE` in the app](https://github.com/in-toto/layout-web-tool/blob/wizard-beta/wizard.py#L76) and override it with `FALSE` in the production [config file](http://flask.pocoo.org/docs/0.12/config/#configuring-from-files).

Reply:
Right now, the default settings (hardcoded in `wizard.py`) don't make a lot of sense because the app fail if no `instance/config.py` is provided. 
My reasoning for letting the app fail was that I didn't want anyone to use the default settings in production without noticing.

Reply:
We probably want to revamp this whole image later to host the mongo daemon/db in a separate container. I'd consider this more of a helpful way to deploy for local testing...

Reply:
leaving DEBUG here. Assuming Dockerfile just for local deployment and testing.